packages/puppeteer-core/src/bidi/core/Browser.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * @license
+ * Copyright 2024 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type * as Bidi from 'chromium-bidi/lib/cjs/protocol/protocol.js';
+
+import {EventEmitter} from '../../common/EventEmitter.js';
+import {throwIfDisposed} from '../../util/decorators.js';
+
+import type BrowsingContext from './BrowsingContext.js';
+import type {SharedWorkerRealm} from './Realm.js';
+import type Session from './Session.js';
+import UserContext from './UserContext.js';
+
+/**
+ * @internal
+ */
+export type AddPreloadScriptOptions = Omit<
+  Bidi.Script.AddPreloadScriptParameters,
+  'functionDeclaration' | 'contexts'
+> & {
+  contexts?: [BrowsingContext, ...BrowsingContext[]];
+};
+
+/**
+ * @internal
+ */
+export default class Browser extends EventEmitter<{
```

Comment:
Default export is bad practice and should be done only with external code - https://google.github.io/styleguide/tsguide.html#imports

Reply:
And https://google.github.io/styleguide/tsguide.html#exports